events/importer/helper/importers/vapaaehtoistyofi/record.py

```diff
@@ -0,0 +1,135 @@
+        if json_dict:
+            self.id = json_dict['id']
+            self.organization_id = json_dict['organization']
+            self.organization_name = json_dict['organizationName']
+            self.title = json_dict['title']
+            self.address = json_dict['address']
+            self.address_coordinates = {
+                'lat': json_dict['addressCoordinates']['lat'],
+                'lon': json_dict['addressCoordinates']['lng']
+            }
+            self.tags = []
+            for theme in json_dict['themes']:
+                theme_id = theme["id"]
+                theme_name = theme["name"]
+                self.tags.append({theme_id: theme_name})
+
+            self.timestamp_start = datetime.datetime.utcfromtimestamp(json_dict['timeStampStartTask'])
+            self.timestamp_end = datetime.datetime.utcfromtimestamp(json_dict['timeStampEndTask'])
+            if json_dict['noActualTime']:
+                self.no_time = True
+            else:
+                self.no_time = False
+            self.description = json_dict['description']
+            self.contact_details = json_dict['contactDetails']
+            self.themes = json_dict['id']
+            self.timestamp_publish = datetime.datetime.utcfromtimestamp(json_dict['timeStampPublicationDate'])
+            # self.publicationTime = json_dict['id']
+            if json_dict['status'] in self.STATUS:
+                self.status = json_dict['id']
+            else:
+                raise RuntimeError("Unknown status %d!" % json_dict['status'])
+            self.creator_id = json_dict['creator']
+            if json_dict['timeStampAdded']:
+                self.timestamp_inserted = datetime.datetime.utcfromtimestamp(json_dict['timeStampAdded'])
+            else:
+                self.timestamp_inserted = None
+            if json_dict['timeStampLastUpdated']:
+                self.timestamp_updated = datetime.datetime.utcfromtimestamp(json_dict['timeStampLastUpdated'])
+            else:
+                self.timestamp_updated = None
+        else:
+            self.id = None
+            self.organization_id = None
+            self.organization_name = None
+            self.title = None
+            self.address = None
+            self.address_coordinates = None
+            self.tags = []
+
+            self.no_time = None
+            self.timestamp_start = None
+            self.timestamp_end = None
+
+            self.description = None
+            self.contact_details = None
+            self.themes = None
+            self.timestamp_publish = None
+
+            self.status = None
+            self.creator_id = None
+            self.timestamp_inserted = None
+            self.timestamp_updated = None
+
+    def get_url_locale(self, locale):
+        if locale not in self.LOCALE:
+            raise ValueError("Unknown locale '%s'!" % locale)
+        return self.TASK_URL_FORMAT % (locale, self.id)
+
+    def __copy__(self) -> object:
```

Comment:
What is the use of this function in the PR?
For copying objects, there are a `copy.copy()` or `copy.deepcopy()` already available in python `copy` module